packages/neon-invoker/src/index.ts

```diff
@@ -78,41 +86,68 @@ export class NeonInvoker implements Neo3Invoker {
 
     const currentHeight = await rpcClient.getBlockCount()
 
-    const trx = new tx.Transaction({
-      script: u.HexString.fromHex(script),
-      validUntilBlock: currentHeight + 100,
-      signers: NeonInvoker.buildMultipleSigner(this.account, cim.signers),
-    })
+    const trx = NeonInvoker.buildTransaction({ account: this.account, script, signers: cim.signers, validUntilBlock: currentHeight + 100 })
 
     const config = {
       ...this.rpcConfig,
       account: this.account,
     }
 
+    const systemFeeOverride = await NeonInvoker.addSystemFeeOverride(config, trx, cim)
+
+    const networkFeeOverride = await NeonInvoker.addNetworkFeeOverride(config, trx, cim, this.account)
+
+    await NeonInvoker.addFees(trx, {
+      ...config,
+      systemFeeOverride,
+      networkFeeOverride,
+    })
+
+    NeonInvoker.signTransaction(trx, this.account, this.rpcConfig.networkMagic)
+
+    return await NeonInvoker.sendTransaction(trx, this.rpcConfig.rpcAddress)
+  }
+
+  static buildTransaction({ script, validUntilBlock, account, signers }: BuildTransaction) {
+    return new tx.Transaction({
+      script: u.HexString.fromHex(script),
+      validUntilBlock,
+      signers: NeonInvoker.buildMultipleSigner(account, signers)
+    })
+  }
+
+  static async addSystemFeeOverride(config: CommonConfig, trx: Neon.tx.Transaction, cim: ContractInvocationMulti) {
```

Comment:
put the trx as the first parameter, the config to the last parameter and change the name of the method to overrideSystemFeeOnTransaction, simply because the name suggest that we are adding something to the instance, which is not the case.